config/bgp_configs.go

```diff
@@ -521,7 +521,7 @@ const (
 	INSTALL_PROTOCOL_TYPE_OSPF               InstallProtocolType = "ospf"
 	INSTALL_PROTOCOL_TYPE_OSPF3              InstallProtocolType = "ospf3"
 	INSTALL_PROTOCOL_TYPE_STATIC             InstallProtocolType = "static"
-	INSTALL_PROTOCOL_TYPE_DIRECTLY_CONNECTED InstallProtocolType = "directly-connected"
+	INSTALL_PROTOCOL_TYPE_DIRECTLY_CONNECTED InstallProtocolType = "connect"
```

Comment:
"directly-connected" comes from OpenConfig:
https://github.com/osrg/public/blob/master/release/models/policy/openconfig-policy-types.yang#L175

Seems needs to fix routeTypeValueMap instead

Reply:
Ok. Let me take a look at routeTypeValueMap. Thanks